### PR TITLE
Added info on RHEL support

### PR DESCRIPTION
**Pull request by crossmi**
Minor updates per findings when getting Ursula working on RHEL 6.7

### CODE REVIEW

**Comment by j2sol** [comment]
This looks fine, but I really don't think we have any intention of supporting Ursula on RHEL6. It's just too old of a platform. RHEL7 would be more appropriate.

**Comment by crossmi** [comment]
The current IBM standard "client" (desktop platform) used for deliver personnel uses RHEL 6.  There isn't a RHEL 7 "Open Client".  Based on our other experience with RHEL 7 though, I don't think any of these items would have issues on RHEL 7...

**Comment by j2sol** [comment]
If attempting to run Urusula on such a platform, I'd push to run it in Docker instead of trying to do it on the native host. In Docker it can be any OS. I don't want to spend time trying to satisfy the needs of random host OS configurations.

**Comment by blueboxjesse** [comment]
:+1: on Jesse's thoughts... Let's focus on containerization vs. base OS implementation.

**Comment by crossmi** [comment]
Would this relate or impact usage of Vagrant though?  E.g. if ursula is run from within a container only, would a host RedHat or other machine be usable for the vagrant VM management in conjunction with running Ursula?

Thanks...

**Comment by j2sol** [comment]
If the vagrant endpoint is reachable, it should be fine.


- jlk

**Comment by j2sol** [comment]
I'm going to close this PR. We don't wand to encourage use of other client operating systems.

**Comment by j2sol** [comment]
Re-opening for further discussion.

File: README.md [hunk 1]
```diff
@@ -45,6 +56,8 @@ depending on where it was installed by pip.
 echo " " >> .bash_profile (for OSX; .bashrc for various linux flavors)
 echo "#sourcing statement for virtualenvwrapper" >> .bash_profile
 echo "source /usr/local/bin/virtualenvwrapper.sh" >> .bash_profile
+# Or the RHEL path is this:
+echo "source /usr/bin/virtualenvwrapper.sh" >> /etc/bashrc  # or ~/.bashrc
```
Comment (craigtracey) [inline README.md]: We should not recommend system-wide. always local to ~/.bashrc only

**Comment by paulczar** [comment]
proposal:

create docs/installing-ursula.md and have a rhel 6 section with these updates,   update README.md pointing the user to this doc for OS specific instructions.

--------------------
# OS Specific Ursula Install Instructions

## RHEL6

blah
blah
blah
---------------------

File: README.md [hunk 2]
```diff
@@ -23,6 +23,17 @@ required to run `ursula`:
...
+
+The following packages are needed on Red Hat Enterprise Linux environments:
+
+* python-pip
+* python-devel
+* python-virtualenvwrapper
+* libxml2-devel
+* libxslt-devel
+* libffi-devel
+
```
Comment (craigtracey) [inline README.md]: Would suggest for all deployments that we recommend updating pip and virtualenv:
```pip install -U pip virtualenv```

**Comment by blueboxjesse** [comment]
@paulczar If we start to do that, who maintains it long term?

File: README.md (hunk 1 above)
Comment (crossmi) [inline README.md]: Okay, yup, I'll tweak.  Thanks!

**Comment by paulczar** [comment]
@blueboxjesse whoever uses that OS and runs across a problem would maintain it by updating it.
I honestly don't think we'll see much beyond `osx`, `ubuntu` and `rhel` in here as that is the two bluebox OS's and the ibm OS.

As much as I am a fan of Docker,  I don't think we're ready to say that that should be the only documented way to run the ursula cli going forward,  certainly most OSes aren't ready for it ( OSX, older linux, Windows, etc ).

Just to make sure we're all on the same page,  we're not talking about installing openstack on an alternative OS,  we're simply talking about documenting how to install and run the ursula cli in the context of being able to deploy a ursula based openstack.

**Comment by blueboxjesse** [comment]
@paulczar Fair enough - just concerned about legacy orphaned docs. But ultimately it's the teams call. :)

**Comment by j2sol** [comment]
The changes you made look okay, but can you squash them into one commit?

**Comment by paulczar** [comment]
we now support centos/rhel in master ... so this PR can be closed.

**Pull request closed without merging**